ops-bedrock/docker-compose.yml

```diff
@@ -194,22 +194,41 @@ services:
       OP_CHALLENGER_NUM_CONFIRMATIONS: 1
 
   da-server:
-    image: us-docker.pkg.dev/oplabs-tools-artifacts/images/da-server:devnet
-    build:
-      context: ../
-      dockerfile: ops/docker/op-stack-go/Dockerfile
-      target: da-server-target
-    command: >
-      da-server
-      --file.path=/data
-      --addr=0.0.0.0
-      --port=3100
-      --log.level=debug
-      --generic-commitment="${PLASMA_GENERIC_DA}"
+    platform: linux/amd64
+    image: ghcr.io/layr-labs/eigenda-proxy@sha256:c7aae98c0ed65763e0bfe9b2f41147123d6d576f435dcc2d0d8dbcbb03cdc205
+    environment:
+      - EIGENDA_PROXY_ADDR=0.0.0.0
+      - EIGENDA_PROXY_PORT=3100
+      - MEMSTORE_ENABLED=true
+      - MEMSTORE_EXPIRATION=45m
+      - EIGENDA_PROXY_SIGNER_PRIVATE_KEY_HEX=0000000000000000000100000000000000000000000000000000000000000000
+      - EIGENDA_PROXY_EIGENDA_DISPERSER_RPC=disperser-holesky.eigenda.xyz:443
     ports:
-      - "3100:3100"
+      - 3100:3100
+      - 6969:7300
+
+  grafana:
+    image: grafana/grafana:11.1.0
+    restart: unless-stopped
+    env_file:
+      -  grafana.env
     volumes:
-      - "da_data:/data"
+      - ./monitoring/grafana/provisioning/:/etc/grafana/provisioning/:ro
+      - ./monitoring/grafana/dashboards:/var/lib/grafana/dashboards
+      # - grafana_data:/var/lib/grafana
+    ports:
+      - 3000:3000
+
+  prometheus:
+    image: prom/prometheus:latest
+    restart: unless-stopped
+    env_file:
+      - .env
```

Comment:
what's in this file? It's missing on my localenv and causing errors. I commented it out and it seems to work, but curious if you had anything valuable in it.